client/my-sites/customer-home/cards/launchpad/index.tsx

```diff
@@ -1,47 +1,53 @@
 import { Button, CircularProgressBar, Gridicon } from '@automattic/components';
-import { updateLaunchpadSettings, useSortedLaunchpadTasks } from '@automattic/data-stores';
+import { useLaunchpadDismisser, useSortedLaunchpadTasks } from '@automattic/data-stores';
 import { Launchpad, type Task } from '@automattic/launchpad';
 import { useTranslate } from 'i18n-calypso';
-import { useEffect, useState } from 'react';
-import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import moment, { Duration } from 'moment';
+import { FC } from 'react';
+import EllipsisMenu from 'calypso/components/ellipsis-menu';
+import PopoverMenuItem from 'calypso/components/popover-menu/item';
 import { useSelector } from 'calypso/state';
 import { getSiteSlug } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import type { AppState } from 'calypso/types';
 
 import './style.scss';
-
 interface CustomerHomeLaunchpadProps {
 	checklistSlug: string;
 	onSiteLaunched?: () => void;
 }
 
-const CustomerHomeLaunchpad = ( {
+const CustomerHomeLaunchpad: FC< CustomerHomeLaunchpadProps > = ( {
 	checklistSlug,
 	onSiteLaunched,
-}: CustomerHomeLaunchpadProps ): JSX.Element => {
+}: CustomerHomeLaunchpadProps ) => {
 	const launchpadContext = 'customer-home';
 	const siteId = useSelector( getSelectedSiteId );
-	const siteSlug = useSelector( ( state: AppState ) => getSiteSlug( state, siteId ) );
-
 	const translate = useTranslate();
-	const [ isDismissed, setIsDismissed ] = useState( false );
+	const siteSlug = useSelector( ( state: AppState ) => getSiteSlug( state, siteId ) || '' );
+
+	const { mutate: dismiss } = useLaunchpadDismisser( siteSlug, checklistSlug );
+
 	const {
-		data: { checklist, is_dismissed: initialIsChecklistDismissed, title },
+		data: { checklist, is_dismissed: isDismissed, is_dismissible: isDismissible, title },
 	} = useSortedLaunchpadTasks( siteSlug, checklistSlug, launchpadContext );
 
-	useEffect( () => {
-		setIsDismissed( initialIsChecklistDismissed );
-	}, [ initialIsChecklistDismissed ] );
-
 	const numberOfSteps = checklist?.length || 0;
 	const completedSteps = ( checklist?.filter( ( task: Task ) => task.completed ) || [] ).length;
 
 	// return nothing if the launchpad is dismissed
 	if ( isDismissed ) {
-		return <></>;
+		return null;
 	}
 
+	const temporaryDismiss = ( duration: Duration ) => {
+		dismiss( {
+			dismissedUntil: moment().add( duration ).utc().unix(),
```

Comment:
I believe we shouldn't be sending the time from the front end. We should be sending `day,`/`week,` and we would do the conversion to the Unix time on the back.

Reply:
Do we have some specific reason for it? 

I think this approach enable us to have more variations with no backend changes, like 2 hours,  8 hours, etc.....

Reply:
Trusting dates coming from the front end can lead to some issues. Even though you made sure we are using UTC, it assumes that the client's time is correct, which may not be true. Another problem I see is the possibility of inputting any other number maliciously, and we'd still calculate it and store it. In this case, it is completely harmless code that takes care of the back end, but we should still prevent such things from happening.

So, I would rather have a string sent to the back with the representation of the time frame, and we then get it translated and validated on the back end.

Reply:
@valterlorran From security perspective, I think both solutions relays to the same problem, we need to validate the input, regardless the date format (string or number). I am going to add validations on the backend, regardless the solution.

Reply:
@valterlorran I updated the frontend and the backend to use relative values. 
* Frontend [4f60c78](https://github.com/Automattic/wp-calypso/pull/86010/commits/4f60c780fa03e03d2c9bbca470f4283e9fb4a7c3)

* Backend https://github.com/Automattic/jetpack/pull/34889/commits/0366b10171fc4ea0683b6f4cf1171a97c52441ac